src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```
@@ -1294,23 +1307,53 @@ protected function initFieldDescriptors()
             'city',
             self::$addressEntityName,
             'contact.address.city',
-            [
-                self::$accountAddressEntityName => new DoctrineJoinDescriptor(
-                    self::$accountAddressEntityName,
-                    $this->getAccountEntityName() .
-                    '.accountAddresses',
-                    self::$accountAddressEntityName . '.main = true', DoctrineJoinDescriptor::JOIN_METHOD_LEFT
-                ),
-                self::$addressEntityName => new DoctrineJoinDescriptor(
-                    self::$addressEntityName,
-                    self::$accountAddressEntityName . '.address'
-                ),
-            ],
+            $accountAddressJoin,
             false,
             true,
             'string'
         );
 
+        $this->fieldDescriptors['zip'] = new DoctrineFieldDescriptor(
+            'zip',
+            'zip',
+            self::$addressEntityName,
+            'contact.address.zip',
+            $accountAddressJoin,
+            false,
+            false,
+            'string'
+        );
+
+        $this->fieldDescriptors['state'] = new DoctrineFieldDescriptor(
```

should not be default selected and also be able to hide (in list)